packages/apk.go

```diff
@@ -26,9 +26,9 @@ type Apk struct {
 	ProjectList parse.ProjectList
 }
 
-func (a Apk) ExtractPurlsFromProjectList(operating string) (purls []string) {
+func (a Apk) ExtractPurlsFromProjectList() (purls []string) {
 	for _, s := range a.ProjectList.Projects {
-		var purl = fmt.Sprintf("pkg:alpine/%s@%s", s.Name, s.Version)
+		var purl = fmt.Sprintf("pkg:apk/alpine/%s@%s", s.Name, s.Version)
```

Comment:
Did we ever get confirmation that this is fine to change these paths??

I think if im reading this right. We were using the old "backward compatible" format and this meets the new format. 
https://ossindex.sonatype.org/doc/coordinates

Reply:
I wanted to get feedback on which way we want to go with this. This was a "read by refactor" moment.  I followed the notes in the current spec and confirmed tests pass but not sure if we want to decouple this.

We also discussed debian being hard coded, so it might be worth fleshing out the format update + os auto detection in another PR. I'm not really sure if we want vulns reported specifically as "debian", "ubuntu", etc. (since there is lag between the various distro sources) or keep it generic since they're all "debs".

That's also true for fedora vs centos and other rhel clones...all rpm, but significant skew between when repos get updates. What's the right way to ensure each distro has the most accurate info about their specific vulns?

Would love input from the larger team, and if we want another PR I can use that to kick it off!